packages/fluentui/react-northstar-fela-renderer/test/felaInvokeKeyframesPlugin-test.ts

```diff
@@ -1,35 +1,34 @@
-import felaInvokeKeyframesPlugin from 'src/utils/felaInvokeKeyframesPlugin';
-
-const renderInvokeKeyframes = felaInvokeKeyframesPlugin();
+import { ICSSInJSStyle } from '@fluentui/styles';
+import felaInvokeKeyframesPlugin from '../src/felaInvokeKeyframesPlugin';
```

Comment:
Just out of curiosity why the paths were changed?

Reply:
We don't have (_and should not_) configured aliases under other packages, so we can't start with `src`. As this package hosts only things related to Fela I don't see any reasons to place files under `utils`.